firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -50,12 +51,12 @@
 
   @NonNull
   public InstallationResponse createFirebaseInstallation(
-      long projectNumber,
+      @NonNull String projectID,
       @NonNull String apiKey,
       @NonNull String firebaseInstallationId,
       @NonNull String appId)
-      throws FirebaseInstallationServiceException {
```

Comment:
As I said above, probably keep two different exceptions, one for here, and one for the API main class.